### ISSUE TITLE
Dedicated D-pad

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#102]
```
A dedicated D-pad would make movement especially on larger devices much easier.

```

Original issue reported on code.google.com by `sdeva...@gmail.com` on 21 Dec 2010 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
It can be similar to Zenonia's virtual controller. It should be possible switch 
control type in the game options (current isn't bad). Also it's good idea, when 
virtual pad is displayed on the screen to keep a hero icon not on the center of 
the screen but little to the right/left (depending on which side controller is 
displayed)
```

Original comment by `R3pl...@gmail.com` on 27 Dec 2010 at 2:54

**Comment by GoogleCodeExporter** [comment]
```
And that would help the other issue where we disappear behind the battle info.
```

Original comment by `doug_y...@hotmail.com` on 27 Dec 2010 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
Although being able to turn it on and off would be amazing. I'm not a big fan 
of the virtual d pad. I've tried Zenonia and I end up using the d pad on my 
phone pretty quick. It's just a weird feel.
```

Original comment by `doug_y...@hotmail.com` on 27 Dec 2010 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
It would be nice to have for those phones who don't have a dedicated d-pad. It 
could be made configurable (switch on or off through preferences)
```

Original comment by `dry...@gmail.com` on 21 Jan 2011 at 9:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 1:34
- Added labels: **Type-Enhancement**, **Priority-Low**
- Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
A virtual d-pad is now available as part of r120.

Marking this as ready for next release.
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 12:54
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
